packages/fxa-shared/tags/sentry.js

```diff
@@ -0,0 +1,117 @@
+// Quick and dirty approach for defining critial endpoints. Future improvements:
+//  - Group by service
+//  - Use regex for precise matching
+//  - Make more config like
+//  - Endpoints are auto generated by traffic capture during smoke tests... so some
+//    of the end points that are only hit by manual testing may not have been captured
+//    and still need to be added.
+
+const CRITICAL_ENDPOINTS = [
```

Comment:
Some of these I don't believe are critical endpoints? 

```
'/sockjs-node',
  '/sockjs-node',
  '/tests/bad%20image%20src',
  '/tests/bad%20url',
  '/tests/undefined/v1/account/profile',
  '/.well-known/openid-configuration',

  '/api/auth_status',
  '/api/email_first',
  '/api/oauth',
  '/api/todos/get',
  '/api/logout',
```

Reply:
@vbudhram This was an effort to see if we could make sentry logs more targeted. See jira item FXA-4262 for more info.